syntax = "proto3";

package message;


enum Module {
  REGISTER = 0;
  RESET = 1;
}

message GetVerifyReq {
  Module module = 1;
  string email = 2;
}

message GetVerifyRsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}

service VerifyService {
  rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}


message GetChatServerReq {
  int32 uid = 1;
}

message GetChatServerRsp {
  int32 error = 1;
  string host = 2;
  string port = 3;
  string token = 4;
}

service StatusService {
  rpc GetChatServer (GetChatServerReq) returns (GetChatServerRsp) {}
}